app/components/Sidebar/index.tsx

```diff
@@ -7,10 +9,14 @@ import Divider from '@mui/material/Divider';
 import ListItem from '@mui/material/ListItem';
 import ListItemButton from '@mui/material/ListItemButton';
 import ListItemText from '@mui/material/ListItemText';
+import Link from 'next/link';
 
 const drawerWidth = 240;
 
 export default function Sidebar() {
+
+  
+  const [categories, setCategories] = useState(['Desayuno', 'Pastas', 'Guarniciones', 'Entradas'])
```

Comment:
Remove states for routes; it's better to use utils files or constants for routes

Reply:
mepa q no

Reply:
I didn't say anything, sorry.